### DISCUSSION TITLE
Welcome to DestinationSol Discussions!

### DISCUSSION THREAD

**Discussion by NicholasBatesNZ** [MovingBlocks/DestinationSol#574] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Please note that Discord will remain our primary source of communication. Join us in the **#destination-sol** channel over at [discord.gg/Terasology](https://discord.gg/Terasology)

**Comment by jozefcifre** [comment]
Hello everyone. I really like this game. I've had a lot of fun with it and would like to see it become more popular, I think it has potential.

What I do, well, not much for now, I'm pretty new to GitHub and everything related, but I'm looking to work on developing this game and make it grow more.
